styles/variables-base.less

```diff
@@ -239,54 +248,54 @@
 @agate-panels-panel-padding: inherit;
 @agate-panels-panel-header-padding: @agate-panels-panel-padding;
 @agate-panels-panel-body-padding: @agate-panels-panel-padding;
-@agate-panels-tab-bar-margin: inherit;
-@agate-panels-tab-bg-position: inherit;
-@agate-panels-tab-bg-transform: inherit;
-@agate-panels-tab-border-radius: inherit;
-@agate-panels-tab-group-border-radius: inherit;
-@agate-panels-tab-group-horizontal-transform: inherit;
-@agate-panels-tab-group-margin: inherit;
-@agate-panels-tab-group-padding: inherit;
-@agate-panels-tab-group-position-after-margin: inherit;
-@agate-panels-tab-group-transform: inherit;
+@agate-panels-tab-bar-margin: 0;
+@agate-panels-tab-bg-position: 0;
+@agate-panels-tab-bg-transform: none;
+@agate-panels-tab-border-radius: 0;
+@agate-panels-tab-group-border-radius: 0;
+@agate-panels-tab-group-horizontal-transform: none;
+@agate-panels-tab-group-margin: 0;
+@agate-panels-tab-group-padding: 0;
+@agate-panels-tab-group-position-after-margin: 0;
+@agate-panels-tab-group-transform: none;
 @agate-panels-tab-margin: inherit;
 @agate-panels-tab-padding: inherit;
-@agate-panels-tab-separator-border-width: inherit;
-@agate-panels-vertical-tab-after-padding-right: inherit;
-@agate-panels-vertical-tab-before-right: inherit;
-@agate-panels-vertical-tab-bg-position: inherit;
-@agate-panels-vertical-tab-group-margin: inherit;
-@agate-panels-vertical-tab-group-padding: inherit;
-@agate-panels-vertical-tab-margin: inherit;
+@agate-panels-tab-separator-border-width: 1px;
+@agate-panels-vertical-tab-after-padding-right: 0;
+@agate-panels-vertical-tab-before-right: 0;
+@agate-panels-vertical-tab-bg-position: 0;
+@agate-panels-vertical-tab-group-margin: 0;
+@agate-panels-vertical-tab-group-padding: 0;
+@agate-panels-vertical-tab-margin: 0;
 @agate-panels-vertical-tab-width: inherit;
 
 // Picker
 // ---------------------------------------
-@agate-picker-border-radius: inherit;
-@agate-picker-transform: inherit;
-@agate-picker-label-transform: inherit;
-@agate-picker-font-weight: inherit;
-@agate-picker-font-size: inherit;
-@agate-picker-active-font-size: inherit;
+@agate-picker-border-radius: 0;
+@agate-picker-transform: none;
+@agate-picker-label-transform: none;
+@agate-picker-font-weight: 500;
```

Comment:
Skin-specific, let's set this to the `@agate-base-font-weight` variable and update the skins as needed.